### PR TITLE
fix: bridge page indexing and static generation

### PR DESCRIPTION
**Pull request by mmioana**
# Which Jira task belongs to this PR?

Contributes to https://linear.app/lifi-linear/issue/JUM-986/fix-broken-page-indexing-for-learn-and-bridge-pages

# Why did I implement it this way?

# Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] This pull request is as small as possible and only tackles one problem
- [ ] I have added tests that cover the functionality / test the bug
- [ ] If this changed the API, I have updated the documentation
- [ ] I have provided QA instructions for the feature / fix implemented in this PR (if applicable)
- [ ] I have provided instructions for any environment / deployment changes that this PR needs when merged

## Summary by CodeRabbit

* **New Features**
  * Added a bridge sitemap index with chunked sitemap generation.
  * Bridge pages now return 404 for invalid routes and show updated page metadata reflecting resolved chains/tokens.
* **Bug Fixes**
  * Normalized site URLs to remove trailing slashes.
  * Improved sitemap error handling to return a service-unavailable response on failures.
  * Robots configuration now references the sitemap index.
* **Tests**
  * Added tests for sitemap generation, chunking, URL utilities, and related edge cases.

### CODE REVIEW

File: src/utils/bridge/resolveBridgeRoute.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      destinationTokenSymbolParam,
+      sourceChain,
+      sourceToken,
+      destinationChain,
+      destinationToken,
+      chains,
+      tokens,
+    };
+  } catch {
+    return null;
```
[the remark this answers is not in the document]
Comment (mmioana) [reply src/utils/bridge/resolveBridgeRoute.ts:88]: Same as for earn

**Review by oktapodia — APPROVED** [review]
